scripts/package.json

```diff
@@ -10,8 +10,8 @@
     "docu:container:generate:container-api": "documentation readme ../container/typings/LuigiContainer.svelte.d.ts --parse-extension ts -f md --readme-file=../docs/luigi-container-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
     "docu:container:generate:compound-container-api": "documentation readme ../container/typings/LuigiCompoundContainer.svelte.d.ts --parse-extension ts -f md --readme-file=../docs/luigi-compound-container-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
     "docu:client": "npm run docu:client:validate && npm run docu:client:generate:section",
-    "docu:client:generate:section": "documentation readme ../client/src/luigi-client.js -f md --readme-file=../docs/luigi-client-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
-    "docu:client:validate": "documentation lint ../client/src/luigi-client.js",
+    "docu:client:generate:section": "documentation readme ../client/public/luigi-client.js -f md --readme-file=../docs/luigi-client-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
+    "docu:client:validate": "documentation lint ../client/public/luigi-client.js",
```

Comment:
I think this part is not quite correct here. **/client/public/luigi-client.js** is a bundled JS file. I think you meant here **luigi-client.ts** here